dispatcher/backend/src/common/schemas/offliners/devdocs.py

```diff
@@ -0,0 +1,162 @@
+from marshmallow import fields
+
+from common.schemas import SerializableSchema, String
+from common.schemas.fields import (
+    validate_output,
+    validate_zim_description,
+    validate_zim_longdescription,
+)
+
+
+class DevDocsFlagsSchema(SerializableSchema):
+    class Meta:
+        ordered = True
+
+    all_flag = fields.Boolean(
+        truthy=[True],
+        falsy=[False],
+        metadata={
+            "label": "All",
+            "description": "Fetch all Devdocs resources, and produce one ZIM "
+            "per resource.",
+        },
+        data_key="all",
+    )
+
+    slug = String(  # should be ListOfString but not yet supported by Zimfarm
+        metadata={
+            "label": "Slug",
+            "description": "Fetch the provided Devdocs resource. "
+            "Slugs are the first path entry in the Devdocs URL. "
+            "For example, the slug for: `https://devdocs.io/gcc~12/` is `gcc~12`.",
+        },
+    )
+
+    first = fields.Integer(
```

Comment:
I see no such option on the website. Or does `DevDocs UI` here means the ZIM UI? Feels like a dev option that shouldn't be on ZF

Reply:
I agree I see no usage for this in production.

Reply:
Removed